kolibri/utils/tests/helpers.py

```diff
@@ -21,7 +24,17 @@ def __init__(self, section, key, value):
 
     def enable(self):
         self.old_value = conf.OPTIONS[self.section][self.key]
+        envvars = option_spec[self.section][self.key]["envvars"]
+        self.old_envvars = {}
+        for envvar in envvars:
+            self.old_envvars[envvar] = os.environ.get(envvar, None)
+            os.environ[envvar] = self.temp_value
```

Comment:
`os.environ` are strings, it should be 
`os.environ[envvar] = str(self.temp_value)`
This is making some tests fail

Reply:
Ah yep - good catch!

Reply:
Updated!